pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/KitUploadRoute.java

```diff
@@ -528,6 +537,10 @@ private String userExistsInRealm(DDPInstance ddpInstanceByRealm, Map<String, Str
         return checkKitUploadNameMatchesToEsName(participantFirstNameFromDoc, participantLastNameFromDoc, participantByShortId);
     }
 
+    private boolean isRgpUpload(DDPInstance ddpInstanceByRealm) {
```

Comment:
Do we have PW coverage for RGP uploads?  If not, do we need a unit test to cover this scenario?

Reply:
I believe we have them for rgp kits, which gives me relief

Reply:
We have PW coverage for the following RGP uploads:

- blood and rna for the proband/main study participant
- blood and rna for the proband's family member

<img width="670" alt="Screenshot 2024-06-18 at 9 57 40 AM" src="https://github.com/broadinstitute/ddp-study-server/assets/5404082/377c44b1-31cd-4417-8660-dab7b9a92c53">